cprover_bindings/src/goto_program/symbol.rs

```diff
@@ -157,6 +167,7 @@ impl Symbol {
             is_lvalue: false,
             is_parameter: false,
             is_static_lifetime: false,
+            is_static_const: false,
```

Comment:
are all _const_ necessarily _static_ ? we already have a flag for _static_lifetime_

Reply:
We are only marking static constants for now. If we were to start marking variables constants, then we could rename this.